### PR TITLE
Enhancement/bottom bar design

### PR DESCRIPTION
**Pull request by poonam-ghewande**
### Summary 
Update bottom tab bar on home page

### Issue ticket number and link    
https://github.com/credebl/adeya-wallet/issues/156

### Checklist before requesting a review
- [x]  All commits contain a DCO Signed-off-by line (we use the DCO GitHub app to enforce this).
- [x]  Self-review of code & No unnecessary commented-out code.
- [x]  All existing functionality remain intact.
- [x]  Updated the README if necessary..
- [x]  No hard-coded sensitive information.

### CODE REVIEW

File: app/theme.ts [hunk 1]
```diff
@@ -578,7 +589,7 @@ export const TabTheme = {
     alignItems: 'center',
   },
   tabBarActiveTintColor: ColorPallet.brand.primary,
-  tabBarInactiveTintColor: ColorPallet.notification.infoText,
+  tabBarInactiveTintColor: '#000',
```
Comment (sairanjit) [inline app/theme.ts:592]: Use colors from `ColorPallet`

**Review by sairanjit — CHANGES_REQUESTED** [review]

File: app/theme.ts [hunk 2]
```diff
@@ -563,11 +574,11 @@ export const ListItems = StyleSheet.create({
 
 export const TabTheme = {
   tabBarStyle: {
-    height: 60,
-    backgroundColor: '#D3E4FA',
-    shadowOffset: { width: 0, height: -3 },
-    shadowRadius: 6,
-    shadowColor: ColorPallet.grayscale.black,
+    height: 80,
+    backgroundColor: '#ffff',
```
[the remark this answers is not in the document]
Comment (poonam-ghewande) [reply app/theme.ts]: done

File: app/theme.ts (hunk 1 above)
Comment (poonam-ghewande) [reply app/theme.ts]: done

**Review by sairanjit — APPROVED** [review]
LGTM

**Pull request merged**